pyvista/plotting/render_window_interactor.py

```diff
@@ -58,14 +64,18 @@ def add_key_event(self, key, callback):
     def add_observer(self, event, call):
         """Add an observer."""
         call = partial(try_callback, call)
-        self._observers[event] = self.AddObserver(event, call)
+        self._observers[event] = self.interactor.AddObserver(event, call)
 
     def remove_observer(self, event):
         """Remove an observer."""
         if event in self._observers:
-            self.RemoveObserver(event)
+            self.interactor.RemoveObserver(event)
             del self._observers[event]
 
+    def remove_observers(self):
+        for obs in self._observers.values():
+            self.interactor.RemoveObservers(obs)
```

Comment:
Don't you also have to update the `self._observers` logistics? Perhaps calling `self.remove_observer` in the loop would be better?